docs/proposal.md

```diff
@@ -2,7 +2,17 @@
 
 ## Motivation and Purpose
 
-TBC
+Our role: Data Analysts at a consulting firm
+
+Target Audience: Movie enthusiasts & Directors at Movie Production Companies
+
+Movie enthusiasts:
+
+Millions of movies are published on a variety of channels; however, filtering excellent movies could be time-consuming. If we could provide a list of popular movies based on customers' preference in genres and productions years, they will save a large amount of time on searching. To solve this challenge, we propose building a data visualization app that allows movie enthusiasts to visually present the top 10 ranking movies in a given year range or genre. Our app will visualize the selection process by providing a slider for the year range and dropdowns for genres. Eventually, detailed information (ranking, title, duration, MPAA ratings) of the top 10 movies will be rendered in a table format. 
```

Comment:
Great work! Only have one suggestion on the last few sentences, we could make it less technical and mention about something like this app can be served as a great exploratory tool for people who like visualization and friendly-user-interface to extract various data at the same time with minimum efforts to select the filters.

"
...to visually present the top 10 ranking movies in a given year range or genre. Our app will visualize the selection process by providing a slider for the year range and dropdowns for genres. Eventually, detailed information (ranking, title, duration, MPAA ratings) of the top 10 movies will be rendered in a table format. "

Reply:
Thanks, I have incorporated your suggestion into the proposal!